SyncIntegrationTests/xcodebuild.py

```diff
@@ -10,7 +10,7 @@
 
 class XCodeBuild(object):
     binary = 'xcodebuild'
-    destination = 'platform=iOS Simulator,name=iPhone 11'
+    destination = 'platform=iOS Simulator,name=iPhone 14 Pro Max'
```

Comment:
Should we use `iPhone 14 Pro Max` instead of `iPhone 14` for the SyncIntegration tests? `iPhone 14 Pro Max` is used in `bitrise.yml`.